genesis.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
We should not put genesis file and config.toml under root folder of the repo, put then under protocol/tests